geetools/ee_image.py

```diff
@@ -1246,17 +1246,34 @@ def tasseledCap(self) -> ee.Image:
     def matchHistogram(
         self,
         target: ee.Image,
-        bands: dict[str, str],
+        bands: Dict[str, str],
```

Comment:
I think the Python version on your end is too old, the dict version should be working on all the Python version after 3.10 (included)